Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -38,7 +38,7 @@ final class ProjectGenerator: ProjectGenerating {
     let configGenerator: ConfigGenerating
 
     /// Generator for the project schemes.
-    let schemesGenerator: SchemesGenerating
+    let schemesGenerator: SchemesGenerator
```

Comment:
Why are we changing the property to depend upon the implementation instead of the abstraction?

Reply:
This was in error, will change it back in the follow up PR.